nmdc_runtime/site/graphs.py

```diff
@@ -112,6 +113,21 @@ def ensure_alldocs():
     materialize_alldocs()
 
 
+@graph
+def run_envo_ontology_load():
+    load_ontology()
+
+
+@graph
+def run_uberon_ontology_load():
+    load_ontology()
+
+
+@graph
+def run_po_ontology_load():
+    load_ontology()
```

Comment:
Do we need three separate `@graph`s here since they're only differentiated by their `op` config?